src/firmware/main/control/control.c

```diff
@@ -2,12 +2,12 @@
 
 #include <math.h>
 
+#include "firmware/main/shared/physics.h"
 #include "io/adc.h"
 #include "io/dr.h"
 #include "io/dsp.h"
 #include "io/encoder.h"
 #include "io/wheels.h"
```

Comment:
```suggestion
#include "firmware/main/io/wheels.h"
```
same with above

Reply:
We need to change a lot of these all over the place, going to say that's out-of-scope for this PR.